website/docs/api-layers.md

```diff
@@ -1267,21 +1267,25 @@ https://github.com/explosion/thinc/blob/master/thinc/layers/padded2list.py
 
 <inline-list>
 
-- **Input:** <tt>Sequence[Any]</tt>
+- **Input:** <tt>Union[Sequence[Hashable], Ints1d, Ints2d]</tt>
 - **Output:** <ndarray>Ints2d</ndarray>
 
 </inline-list>
 
-Remap string or integer inputs using a mapping table, usually as a preprocess
-before embeddings. The mapping table can be passed in on input, or updated after
-the layer has been created. The mapping table is stored in the `"mapping_table"`
-attribute.
+Remap a sequence of strings, integers or other Hashable inputs using a
+mapping table, usually as a preprocessing step before embeddings. The
+input can also be a two dimensional integer array in which case the
+`column: int` attribute tells the `remap_ids` layer which column of the
+array to map. For 2D inputs the `column` has to be set on initialization.
+The mapping table can be set at initialization, but can also be passed in on input, 
+When provided on initializtion the mapping table is stored in the `"mapping_table"`
+attribute. 
```

Comment:
Aside from the spelling error ("initializtion"), I don't understand the point that you're trying to make here? It's always stored in this attribute, right?

Reply:
Oh right that's true. But it can also be passed later, like its not going to throw an error basically if you want to switch it later on. Do you have some suggestion about how to say that?

Reply:
I think it would be clearer to mention that `column` and `mapping_table` can be modified after initialization. Something about how it's set up to be flexible in terms of how the mapping is configured so that the values matter at the point where you call `forward` and not in `init` (but I wouldn't phrase it exactly like that).